packages/http/src/mocker/__tests__/functional.spec.ts

```diff
@@ -25,20 +24,18 @@ describe('http mocker', () => {
         expect(response).toMatchSnapshot();
       });
 
-      test('and that content type does not exist should throw', () => {
-        const rejection = mocker.mock({
-          resource: httpOperations[0],
-          input: httpRequests[0],
-          config: {
-            mock: {
-              mediaType: 'text/funky',
+      test('and that content type does not exist should return empty body', () => {
+        return expect(
+          mocker.mock({
+            resource: httpOperations[0],
+            input: httpRequests[0],
+            config: {
+              mock: {
+                mediaType: 'text/funky',
+              },
             },
-          },
-        });
-
-        return expect(rejection).rejects.toEqual(
-          new Error('Requested content type is not defined in the schema')
-        );
+          })
+        ).resolves.toMatchObject({ headers: { 'Content-type': 'text/plain' } });
```

Comment:
Let's match the response body too. I'd like us to be explicit about it.

Reply:
Done in d6dc96f